src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -38,36 +37,25 @@ object Command {
     }
   }
 
-  def run(implicit ps: Parameters): Try[Unit] = {
-    for {
-      records <- parse(ps.input)
-      _ <- records.map(update).find(_.isFailure).getOrElse(Success(())) // fail fast, if an error occurs, stop updating the rest of the stream!
-    } yield records.size
-  }
+  def run(implicit ps: Parameters): Try[Unit] = for {
+    records <- parse(ps.input)
+    _ <- records.map(update).find(_.isFailure).getOrElse(Success(())) // fail fast, if an error occurs, stop updating the rest of the stream!
+  } yield records.size
```

Comment:
In test mode you might not (always) want to fail fast when the transformations detect problems with the content of a stream of a particular dataset.